jenkins/Jenkinsfile

```diff
@@ -326,7 +353,15 @@ timestamps {
                             uploadFiles ("package/manifest", "jenkins/${branchName}/latest")
 
                             manifestFilename = "manifest"
-                            artifactsFilename = "artifacts.json"
+                        }
+
+                        // Generate, Upload and Report artifacts.json
+                        stage('report-artifacts') {
```

Comment:
We already have quite a few stages, which takes up a significant amount of horizontal space in the ocean blue UI. Could the new stages you've added just be included in the "Upload to Azure" stage, thus not creating any new stages?